package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I am not sure why `react-onclickoutside` is the only dependency showing up in your package.json and why it is showing up at this time? We already had this package installed in the past.